### PR TITLE
refactor: /model/Model.ts

### PR DESCRIPTION
**Pull request by WongYC-66**
Create helper file & functions at :
- /src/helpers/ModelHelper.ts

Refactor : 
- /src/model/Model.ts

Test result:
image

### CODE REVIEW

File: src/helpers/ModelHelper.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { BaseModel } from 'src/model/Model';
```
Comment (pockettojs) [inline src/helpers/ModelHelper.ts]: Want to ask: why not add the function directly into the Model, instead, introduce a ModelHelper file?

Comment (WongYC-66) [reply src/helpers/ModelHelper.ts]: That was my first intuition, until i saw the folder of /src/helper/ 

If those functions only used by /model/Model.ts , i agree dont create extra file

Comment (WongYC-66) [reply src/helpers/ModelHelper.ts]: Ok. Latest commit now doesn't have the extra ModelHelper file.

Comment (pockettojs) [reply src/helpers/ModelHelper.ts]: Sure, I will merge now

**Pull request merged**